internal/services/containers/kubernetes_cluster_node_pool_resource.go

```diff
@@ -430,6 +430,23 @@ func resourceKubernetesClusterNodePoolSchema() map[string]*pluginsdk.Schema {
 			Type:     pluginsdk.TypeBool,
 			Optional: true,
 		},
+
+		"gateway_profile": {
```

Comment:
Is it allowed to set this property in the kubernete cluster's default node pool?